doc/devices/blocks.md

```diff
@@ -136,7 +136,7 @@ mot_block = block_mot("motor_block")
 `BlockRw` and `BlockRwRbv` both take a `write_config` argument, which can be used to configure
```

Comment:
(I can't comment on a few lines above but this is referring to the block_mot bit above) 

we should probably just mirror what the docstring says about motors not explicitly needing a wait on their own underneath the "run control" bit as `waitfor_move` is pretty commonly used in the scans library I think.